### PR TITLE
Jostein wip

### PR DESCRIPTION
**Pull request by joakim-hove**
Assembkled here for github viewing pleasure.

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQEnums.cpp [hunk 1]
```diff
@@ -268,6 +268,57 @@ std::string typeName(UDQVarType var_type) {
     }
 }
 
+UDAKeyword keyword(UDAControl control) {
+    if (control == UDAControl::WCONPROD_ORAT)
```
Comment (akva2) [inline src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQEnums.cpp]: this could be a (unordered) map.

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQEnums.cpp]: Yes it could... 😀

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Pull request closed without merging**